components/higher-order/with-global-events/index.js

```diff
@@ -49,15 +48,26 @@ function withGlobalEvents( eventTypesToHandlers ) {
 
 			handleEvent( event ) {
 				const handler = eventTypesToHandlers[ event.type ];
-				if ( typeof this.ref.current[ handler ] === 'function' ) {
-					this.ref.current[ handler ]( event );
+				if ( typeof this.wrappedRef[ handler ] === 'function' ) {
+					this.wrappedRef[ handler ]( event );
+				}
+			}
+
+			handleRef( el ) {
+				this.wrappedRef = el;
+				if ( this.props.forwardedRef ) {
```

Comment:
When will this condition not be satisfied?

Reply:
For some reason when `react-test-renderer` is used this condition is not satisfied.

Reply:
>I know we shouldn't implement logic to satisfy tests, but this was a quick and easy fix.

Please make an issue to correct this, ideally self-assigned.

Reply:
This has been fixed.

Reply:
I don't think it's been fixed; this line is causing #7707 and adding the check back prevents the error.

Though it would indicate these refs aren't being forwarded at all.